### PR TITLE
Fix `_query` function.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: test/web/10-VerifiableCredentialStore.js [hunk 1]
```diff
@@ -115,7 +115,8 @@ describe('VerifiableCredentialStore', () => {
     should.not.exist(credential);
   });
 
-  it('should query for an AlumniCredential with any issuer', async () => {
+  it('should not find credential when querying for an AlumniCredential' +
```
Comment (dlongley) [inline test/web/10-VerifiableCredentialStore.js:118]:
```suggestion
  it('should not find credential when querying for an AlumniCredential ' +
```

Comment (dlongley) [reply test/web/10-VerifiableCredentialStore.js:118]: Shouldn't we keep the existing test and *add* this new one?

File: test/web/10-VerifiableCredentialStore.js [hunk 2]
```diff
@@ -125,11 +126,10 @@ describe('VerifiableCredentialStore', () => {
     const newCred = Object.assign({}, AlumniCredential, {id: 'foo'});
     await vcStore.insert({credential: newCred});
 
+    // query with an issuer that is not the same as the issuer on the credential
     const credentials = await vcStore.match({query: query.query1});
```
Comment (dlongley) [inline test/web/10-VerifiableCredentialStore.js:130]: Can we import `query` as `queries` instead -- and rename `query1` to something more semantic, i.e., something that represents what the query is for / what makes it unique from the other queries in `queries`? Maybe `queries.queryWithTrustedIssuer` or something like that?

**Review by dlongley — CHANGES_REQUESTED** [review]

File: test/web/10-VerifiableCredentialStore.js (hunk 2 above)
Comment (JSAssassin) [reply test/web/10-VerifiableCredentialStore.js]: Updated here 99e929c

File: test/web/10-VerifiableCredentialStore.js (hunk 1 above)
Comment (JSAssassin) [reply test/web/10-VerifiableCredentialStore.js]: I am not sure what you mean. There is already a test for the positive case where we are passing the correct issuer in the query here ---> https://github.com/digitalbazaar/bedrock-web-vc-store/pull/16/files#diff-affd32a517d78f0385d37ce90c3642cfbd3233ce09cbd12ab2f5246e1a132d57R140. This one on L118 is the test for the negative case.

Comment (dlongley) [reply test/web/10-VerifiableCredentialStore.js]: Well, there's a test for one with a correct issuer, but this test used to say "any issuer" -- where `trustedIssuer` is not set. There are three cases to check:

1. Match on `trustedIssuer`.
2. Miss on non-matching `trustedIssuer`.
3. Match on no `trustedIssuer` (any will do).

File: test/web/10-VerifiableCredentialStore.js [hunk 3]
```diff
@@ -125,27 +129,30 @@ describe('VerifiableCredentialStore', () => {
     const newCred = Object.assign({}, AlumniCredential, {id: 'foo'});
     await vcStore.insert({credential: newCred});
 
-    const credentials = await vcStore.match({query: query.query1});
+    // query with an issuer that is not the same as the issuer on the credential
+    const credentials = await vcStore.match({
+      query: queryWithIncorrectTrustedIssuer
```
Comment (dlongley) [inline test/web/10-VerifiableCredentialStore.js:134]: `Incorrect` is confusing -- how can an issuer be "incorrect"? Perhaps `queryWithMatchingTrustedIssuer` and `queryWithNonMatchingTrustedIssuer`?

Comment (JSAssassin) [reply test/web/10-VerifiableCredentialStore.js]: Addressed here f0cd1d0

File: test/web/10-VerifiableCredentialStore.js (hunk 1 above)
Comment (JSAssassin) [reply test/web/10-VerifiableCredentialStore.js]: Added test for the `match on no trustedIssuer` case here f0cd1d0

**Review by dlongley — APPROVED** [review]
Thanks!

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
Thanks!

**Comment by dmitrizagidulin** [comment]
`bedrock-web-vc-store@4.0.1` published.

**Pull request merged**